source/extensions/filters/http/original_src/original_src.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "envoy/http/filter.h"
+#include "envoy/network/address.h"
+
+#include "common/common/logger.h"
+
+#include "extensions/filters/http/original_src/config.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace OriginalSrc {
+
+/**
+ * Implements the Original Src network filter. This filter places the source address of the socket
+ * into an option which will alter be used to partition upstream connections.
+ * This does not support non-ip (e.g. AF_UNIX) connections, which will be failed and counted.
```

Comment:
Looks like they are actually just ignored based on the code? ie we dont add any socket options but let is pass through

Reply:
I'll update this to be clear.